docs/install/requirements.md

```diff
@@ -15,13 +15,15 @@ InvenioRDM depends on the following requirements to be installed on your local s
         SYSTEM_VERSION_COMPAT=1 invenio-cli install
         ```
 
+    - In case that `invenio-cli` (and other commands installed via `pip`) cannot be found after installing, you may have to update your `$PATH` to include the install directory (e.g. `PATH="$PATH:$HOME/.local/bin"` on Linux).
 
+- [Pipenv](https://pipenv.pypa.io/en/latest/) and [pyenv](https://github.com/pyenv/pyenv#installation) (or [asdf](https://asdf-vm.com/) with [asdf-python](https://github.com/danhper/asdf-python)) for managing virtual environments. Note the [requirements](https://github.com/pyenv/pyenv/wiki#suggested-build-environment) for both cases!
```

Comment:
I think we can skip this paragraph. `pipenv` is a setup.py requirement to `invenio-cli` so it gets installed when invenio-cli is installed. (asdf wouldn't be enough given that invenio-cli uses pipenv under the hood anyway).